community/mailinglists.md

```diff
@@ -0,0 +1,34 @@
+---
+breadcrumb: Mailing Lists
+---
+# Mailing Lists
+
+Here are the [mailing lists](https://mta.openssl.org/) we run. You must
+be a member of the list to post to it. Anything you post to a list,
+including the E-mail address you posted from, will be sent to and seen
+by all other members of the list.
+
+| List                                                                          | Purpose                                                                           |
+|-------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|
+| [openssl-announce](https://mta.openssl.org/mailman/listinfo/openssl-announce) | Official Project Announcements; low-volume read-only.                             |
+| [openssl-project](https://mta.openssl.org/mailman/listinfo/openssl-project)   | Discussion about the organization and structure of the project itself. Moderated. |
+| [openssl-commits](https://mta.openssl.org/mailman/listinfo/openssl-commits)   | Commits to the source repository and build results; read-only.                    |
+| [openssl-dev](https://mta.openssl.org/mailman/listinfo/openssl-dev)           | This is now a read-only list, and is archived.                                    |
+| [openssl-users](https://mta.openssl.org/mailman/listinfo/openssl-users)       | Any questions about building, using, etc., OpenSSL itself.                        |
```

Comment:
I see similar glitches with this table on beta.openssl.org as on the mirrors.md table. I suppose you can just delete the line 12 to avoid the extra borderline under the openssl-announce line. However it is still merged with the text below and similarly for the archives table.

Reply:
Perfect!